apps/studio/lib/api/apiWrapper.test.ts

```
@@ -0,0 +1,148 @@
+
+describe('apiWrapper', () => {
+  const mockReq = {} as any
+  const mockRes = {
+    status: vi.fn().mockReturnThis(),
+    json: vi.fn().mockReturnThis(),
+  } as any
+  const mockHandler = vi.fn()
+
+  beforeEach(() => {
+    vi.clearAllMocks()
+    ;(IS_PLATFORM as any) = true
```

Biome (2.1.2)

[error] The imported variable IS_PLATFORM is read-only

The variable is imported here

Use a local variable instead of reassigning an import.

(lint/suspicious/noImportAssign)